eng/Versions.props

```diff
@@ -118,7 +118,6 @@
     <XUnitVersion>2.4.1</XUnitVersion>
     <TraceEventVersion>2.0.5</TraceEventVersion>
     <NewtonsoftJsonVersion>12.0.3</NewtonsoftJsonVersion>
-    <XUnitXmlTestLoggerVersion>2.1.26</XUnitXmlTestLoggerVersion>
```

Comment:
Why is this removed? Just left-over?

Reply:
Yeah this was introduced only for vstest in a past PR but actually we don't need it.